src/main.rs

```diff
@@ -15,14 +17,25 @@ mod state;
 
 static FONT: &[u8] = include_bytes!("../assets/fonts/NotoSans/NotoSans-Regular.ttf");
 
+#[derive(Debug, StructOpt)]
+#[structopt(name = "spacerun")]
+pub struct Options {
+    #[structopt(short = "k", long = "key", parse(try_from_str))]
```

Comment:
To keep it consistent with the new config we should name the arg "shortcut" imo

Reply:
yeaah ok you're right ;)